src/seedu/addressbook/data/person/Person.java

```diff
@@ -28,8 +26,9 @@
     private Fees fees;
     private final Set<Tag> tags = new HashSet<>();
     private final Set<Exam> exams = new HashSet<>();
+    private final Set<Assessment> assessments = new HashSet<>();
     private Attendance attendance;
```

Comment:
Maybe an improvement would be to use java SortedSet, to keep the Assessments sorted when listed.